libs/utils/src/tabulation/tabulation.test.ts

```diff
@@ -37,47 +37,59 @@ import {
   getScannedBallotCount,
   getHmpbBallotCount,
 } from './tabulation';
-import { CAST_VOTE_RECORD_REPORT_FILENAME } from '../filenames';
 import {
   convertCastVoteRecordVotesToTabulationVotes,
   getCurrentSnapshot,
 } from '../cast_vote_records';
 
-/**
- * For testing with small cast vote record files only.
- */
-function loadCastVoteRecordsFromReport(
-  directoryPath: string
+function castVoteRecordToTabulationCastVoteRecord(
+  castVoteRecord: CVR.CVR
+): Tabulation.CastVoteRecord {
+  return {
+    ballotStyleId: castVoteRecord.BallotStyleId,
+    batchId: castVoteRecord.BatchId,
+    card: castVoteRecord.BallotSheetId
+      ? // eslint-disable-next-line vx/gts-safe-number-parse
+        { type: 'hmpb', sheetNumber: Number(castVoteRecord.BallotSheetId) }
+      : { type: 'bmd' },
+    partyId: castVoteRecord.PartyIds?.[0],
+    precinctId: castVoteRecord.BallotStyleUnitId,
+    scannerId: castVoteRecord.CreatingDeviceId,
+    votes: convertCastVoteRecordVotesToTabulationVotes(
+      assertDefined(getCurrentSnapshot(castVoteRecord))
+    ),
+    votingMethod: castVoteRecord.vxBallotType,
+  };
+}
+
+function readCastVoteRecordExport(
+  exportDirectoryPath: string
 ): Tabulation.CastVoteRecord[] {
-  const cvrReport = safeParse(
-    CVR.CastVoteRecordReportSchema,
-    JSON.parse(
+  const castVoteRecordIds = readdirSync(exportDirectoryPath, {
+    withFileTypes: true,
+  })
+    .filter((entry) => entry.isDirectory())
+    .map((directory) => directory.name)
+    .sort();
```

Comment:
Here you can see the shift from the old CVR format (one big `cast-vote-record-report.json`) to the new one (a subdirectory for every CVR)